Extension/package.json

```diff
@@ -6322,7 +6322,7 @@
         "@typescript-eslint/eslint-plugin": "^6.1.0",
         "@typescript-eslint/parser": "^6.1.0",
         "eslint-plugin-header": "^3.1.1",
-        "@vscode/test-electron": "^2.3.3",
+        "@vscode/test-electron": "^2.3.8",
```

Comment:
A bug in the previous version was breaking the download of the vscode test packages.